### PR TITLE
Auto create basic tier

### PR DESCRIPTION
**Pull request by Dannyps**

### CODE REVIEW

File: Backbone.API/ApplicationDbContextSeed.cs [hunk 1]
```diff
@@ -58,4 +60,14 @@ private async Task SeedApplicationUsers(DevicesDbContext context)
 
         await context.SaveChangesAsync();
     }
+
+    private async Task SeedBasicTier(DevicesDbContext context)
+    {
+        if(await context.Tiers.AnyAsync())
+            return;
+
+        await context.Tiers.AddAsync(new Tier(TierName.Create("Basic Tier").Value));
```
Comment (tnotheis) [inline Backbone.API/ApplicationDbContextSeed.cs:69]: The Tier name shouldn't contain the word "Tier". That's redundant.

**Review by tnotheis — CHANGES_REQUESTED** [review]
There is something wrong with this PR. There are lots of changes in here that @daniel-almeida-konkconsulting made on another branch.

**Comment by Dannyps** [comment]
This branch was based on `feature/create-tier--create-tier-command` but that branch does not seem to exist anymore, so I set `feature/create-tier` as base instead.

**Comment by tnotheis** [comment]
Take a look at the diff. There is some pipeline stuff listed that shouldn't be there

**Comment by Dannyps** [comment]
Sorry for the force, I couldn't revert a merge commit that was poisioning the PR. It should be good now.

**Review by tnotheis — CHANGES_REQUESTED** [review]
Just noticed that there is a naming error on the APplicationDbContextSeed class, which originates from the microservice times.
The class should be called `DevicesDbContextSeed`. Could you fix that please?

**Review by tnotheis — APPROVED** [review]

**Pull request merged**